pkg/check/util.go

```diff
@@ -103,3 +106,56 @@ func getResourcePool(ctx *CheckContext, ref vim.ManagedObjectReference) (*mo.Res
 
 	return &resourcePoolMo, resourcePoolPath, nil
 }
+
+// getClusterComputeResource returns the ComputeResource that matches the provided name.
+func getClusterComputeResource(ctx *CheckContext, computeCluster string, datacenter *object.Datacenter) (*object.ClusterComputeResource, error) {
+	klog.V(4).Infof("Looking for CC: %s", computeCluster)
+	tctx, cancel := context.WithTimeout(ctx.Context, 60*time.Second)
```

Comment:
Is there any reason to not use `Timeout` variable in these calls? We would like to control the calls to vCenter via CLI configuration wherever possible.